Taskfile.yml

```diff
@@ -24,4 +24,13 @@ tasks:
   run:
     desc: Run the Weather Data Fetcher
     cmds: 
-      - poetry run python weather_data_fetcher.py
\ No newline at end of file
+      - poetry run python weather_data_fetcher.py
+
+  test:
+    desc: Runs tests on the code
+    cmds:
+      - >
+        poetry run pytest
+        -s
```

Comment:
Pleas write out command line options if you can. I have no fricking clue what `-s` is :question: